lib/base/RequestClient.js

```diff
@@ -107,4 +111,33 @@ RequestClient.prototype.request = function(opts) {
   return deferred.promise;
 };
 
+function debugLogRequest(options) {
+  const STANDARD_HEADERS = ['user-agent', 'accept-charset', 'connection', 'authorization', 'accept', 'content-type'];
+
+  console.log('-- BEGIN Twilio API Request --');
+  console.log(`${options.config.method} ${options.config.url}`);
+
+  if (options.data) {
+    console.log('Form data:');
+    console.log(options.data);
+  }
+
+  if (options.params && Object.keys(options.params).length > 0) {
+    console.log('Querystring:');
+    console.log(options.params);
+  }
+
+  const customHeaders = Object.keys(options.headers).filter((header) => {
+    return !STANDARD_HEADERS.includes(header.toLowerCase());
```

Comment:
I don't think we need to remove standard headers. They could be helpful for debugging.